src/input/base-input.js

```diff
@@ -230,12 +244,21 @@ class BaseInput<T: EventTarget> extends React.Component<
       >
         <ClearIcon
           size={16}
+          tabindex={0}
           title={ariaLabel}
           aria-label={ariaLabel}
           onClick={this.onClearIconClick}
+          onKeyUp={event => {
+            if (event.key && (event.key === 'Enter' || event.key === ' ')) {
+              this.onClearIconClick();
```

Comment:
you likely need `event.preventDefault()` here to avoid scrolling the page on `space`

Reply:
also needed to change to keyDown